docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -379,8 +379,8 @@ All smart contracts that you create with the zkApp CLI use similar code:
 Comments break down each stage:
 
 ```ts src/main.ts
-19 const { privateKey: deployerKey, publicKey: deployerAccount } = Local.testAccounts[0];
-20 const { privateKey: senderKey, publicKey: senderAccount } = Local.testAccounts[1];
+19
```

Comment:
@Saurabhpatil-dev  
To verify this code update, please make the corresponding change in the example code file https://github.com/o1-labs/docs2/blob/main/examples/zkapps/01-hello-world/src/main.ts

Reply:
Hi @barriebyron ,

I compared both of the files and it seems the one which are asking me to update is already correct and matching to the changes I committed.

I am attaching the screenshot for your reference:

The file on **_left side is the one I have committed_** and the **_another one which you are asking me to update is on right side_**.  
The duplicated code was removed by me from the left file on line number 19-20. 
But when checked on the file in right side the duplicate code is already removed. So, I don't think so we need to update it. 
Please let me know if you would like me to update the file to match the line numbers in both files or make any other changes.

![Screenshot 2023-11-22 at 10 32 27 PM](https://github.com/o1-labs/docs2/assets/55076843/eea36787-832b-49ca-8f0d-a46e04c9bf1d)

Thank you 😃